src/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/GetDocumentById.cs

```diff
@@ -37,7 +39,7 @@ NpgsqlTransaction transaction
 
         try
         {
-            JsonNode? edfiDoc = await _sqlAction.FindDocumentEdfiDocByDocumentUuid(
+            var document = await _sqlAction.FindDocumentEdfiDocByDocumentUuid(
```

Comment:
Please don't switch to `var` when there is already a type. It took me a bit to figure out the type actually changed.